project/project/CakePlatformDependencies.scala

```diff
@@ -113,7 +113,6 @@ object CakePlatformDependencies {
     val packager: ModuleID =
```

Comment:
The akka-http template is using spray-json, so it sounds like we can drop the playJson dependency (assuming that the play template has no need of it).

janino is currently only used in play template for colouring. Again, akka-http is not using this, so perhaps we should move this to the play template?